src/AppBundle/Entity/Tag.php

```diff
@@ -26,6 +26,13 @@ class Tag implements \JsonSerializable
     private $id;
 
     /**
+     * @var array
+     *
+     * @ORM\ManyToMany(targetEntity="AppBundle\Entity\Post", mappedBy="tags")
+     */
+    private $posts;
```

Comment:
I agree with @yceruto.  There is a Doctrine best practice:

> Avoid bidirectional relationships. 

http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/best-practices.html#constrain-relationships-as-much-as-possible